### PR TITLE
Check the limitation of the number of hosts to be terminated at one time when termnating a host gracefully

### PR DESCRIPTION
**Pull request by liyaqin1**
Check the limitation of the number of hosts to be terminated at one time when termnating a host gracefully

### CODE REVIEW

**Comment by liyaqin1** [comment]
tested on dev1.
if the termination is set to 1, and if we terminate one host and then immediately terminate another one, we got a warning:
Screenshot 2023-04-24 at 4 38 33 PM
Screenshot 2023-04-24 at 4 38 46 PM

File: deploy-board/deploy_board/templates/host_side_panel.html [hunk 1]
```diff
@@ -42,6 +46,44 @@ <h4 class="modal-title">Host Termination Confirm</h4>
...
+                return result.length;
+            } catch (error) {
+                console.error(error);
+                return 0;
+            }
+        };
+        doAjax().then( (data) => {
+            terminatingHostCount = data;
+            //console.log(terminatingHostCount);
+            //console.log(`{{termination_limit}}`);
```
Comment (tylerwowen) [inline deploy-board/deploy_board/templates/host_side_panel.html:79]: remove

File: deploy-board/deploy_board/templates/host_side_panel.html [hunk 2]
```diff
@@ -42,6 +46,44 @@ <h4 class="modal-title">Host Termination Confirm</h4>
         </div><!-- /.modal-content -->
     </div><!-- /.modal-dialog -->
 </div><!-- /.modal -->
+
+<script>
+    $("#terminateBtn").click(function () {
+        let element = document.getElementById("terminationWarning");
+        // Clean up first
+        element.setAttribute("hidden", "hidden");
```
Comment (tylerwowen) [inline deploy-board/deploy_board/templates/host_side_panel.html:54]: You can set element to hidden by default in HTML

File: deploy-board/deploy_board/templates/host_side_panel.html [hunk 3]
```diff
@@ -42,6 +46,44 @@ <h4 class="modal-title">Host Termination Confirm</h4>
         </div><!-- /.modal-content -->
     </div><!-- /.modal-dialog -->
 </div><!-- /.modal -->
+
+<script>
+    $("#terminateBtn").click(function () {
+        let element = document.getElementById("terminationWarning");
```
[the remark this answers is not in the document]
Comment (liyaqin1) [reply deploy-board/deploy_board/templates/host_side_panel.html:52]: ok

File: deploy-board/deploy_board/templates/host_side_panel.html [hunk 4]
```diff
@@ -42,6 +46,39 @@ <h4 class="modal-title">Host Termination Confirm</h4>
...
+                        xhr.setRequestHeader("X-CSRFToken", csrftoken);
+                    },
+                });
+                return result.length;
+            } catch (error) {
+                console.error(error);
+                return 0;
+            }
+        };
+        doAjax().then( (data) => {
```
Comment (tylerwowen) [inline deploy-board/deploy_board/templates/host_side_panel.html:70]: You don't have to call backend at all if `{{termination_limit}} === null`.
Indeed you don't even have to render the UI if `termination_limit` is not set.

**Review by tylerwowen — APPROVED** [review]

**Pull request merged**